guide/portuguese/javascript/html-dom/index.md

```diff
@@ -1,16 +1,14 @@
 ---
-title: HTML Dom
-localeTitle: HTML Dom
+title: HTML DOM (Document Object Model)
```

Comment:
Please verify the Title section in the [guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#title): Article titles should be as short, concise, and as to-the-point as possible.
```suggestion
title: HTML DOM (Document Object Model)
```

Reply:
@CleoAguiar  Actually, the title should be the same as the filename.  If everything else looks good here, then we can merge this one.

@paiva-thiago Please change the title back to the original `HTML Dom`

Reply:
@RandellDawson I made a commit with this change. Do I need to do a new pull request for that?

Reply:
@RandellDawson Done!